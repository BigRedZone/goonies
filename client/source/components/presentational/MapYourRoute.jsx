import React from 'react';
import ReactDom from 'react-dom';
import GoogleMapsContainer from '../container/renderMap.jsx';

class MapYourRoute extends React.Component {
  constructor (props) {
    super (props)
  }

  render () {
    const divStyle = {
      border: '1px solid black',
      margin: '20px',
      padding: '50px'
    };
    return (
      <div>
        <div> 
          <GoogleMapsContainer />
        </div>
        <div style={divStyle}> --Map information will go here --
        </div>
      </div>
    )
  }
}

export default MapYourRoute;