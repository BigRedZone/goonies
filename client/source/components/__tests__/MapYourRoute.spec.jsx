import React from 'react';
import renderer from 'react-test-renderer';
import MapYourRoute from '../presentational/MapYourRoute';

test('MapYourRoute Renders correctly', () => {
  const component = renderer.create(<MapYourRoute />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
