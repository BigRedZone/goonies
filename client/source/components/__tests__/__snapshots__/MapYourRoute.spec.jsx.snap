// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MapYourRoute Renders correctly 1`] = `
<div>
  <h3>
    Your Map
  </h3>
  <div>
    <div>
      Loading...
    </div>
  </div>
</div>
`;
